README.md

```diff
@@ -1,12 +1,12 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+ 
+This weeks project was to build a bussines page whit a form and a accordion and a background picture or video, 
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If I had more time I would make the logo spin 360deg, That would be a cool effect I think. I had no big struggle, but the possition rellative and absolute does mather =D I have really learnd that now.
```

Comment:
That would be sooo cool! 
You gave me inspiration to try it out also on some of the future projects :)